packages/pass-style/tools/arb-passable.js

```diff
@@ -1,19 +1,22 @@
 // @ts-check
 import '../src/types.js';
```

Comment:
Uniquely for this js file, github is not doing any syntax highlighting for me. Do you see it highlighted? Any idea why I am not?

Reply:
I’m seeing highlights.